gitlab-cng.yaml

```diff
@@ -54,6 +53,25 @@ subpackages:
             cp -r $x ${{targets.subpkgdir}}/$x
           done
 
+  - name: "${{package.name}}-base"
+    pipeline:
+      - runs: |
+          cd ./gitlab-base
+          for x in $(find scripts/ -type f); do
+            mkdir -p ${{targets.subpkgdir}}/$(dirname $x)
+            cp -r $x ${{targets.subpkgdir}}/$x
+          done
+    dependencies:
```

Comment:
I’d do these at the image level.  Gitlab has several flavors of each of their images and it’s not clear to me that each of them is going to have this full set.  For example, I’m not sure if their hardened image has xtail or procps.

If there are things like perhaps the logger, gomplate, busybox that are truly common across all images I’d just add them to the range version.

fwiw ca-certs is a dependency of our baselayout, so it’s never needed explicitly.

Reply:
I think this would simplify the creation of the images, and it'll also "fix" the creation of a current package `gitlab-cng-base` which wasn't quite useful at this moment. 

It seems xtail and procps seems to be part of their gitlab-base image https://gitlab.com/gitlab-org/build/CNG/-/blob/master/gitlab-base/Dockerfile?ref_type=heads#L27. Also I can confirm procps is used in 8 images, xtail in 5 images, gitlab-logger in all of them, as well as gomplate.

> fwiw ca-certs is a dependency of our baselayout, so it’s never needed explicitly.

I wasn't aware of that detail, I can remove it.